diff --git a/nmxact/nmserial/serial_xport.go b/nmxact/nmserial/serial_xport.go
index a0368e7..a646fe8 100644
--- a/nmxact/nmserial/serial_xport.go
+++ b/nmxact/nmserial/serial_xport.go
@@ -276,7 +276,8 @@ func (sx *SerialXport) Tx(bytes []byte) error {
 			/* slower platforms take some time to process each segment
 			 * and have very small receive buffers.  Give them a bit of
 			 * time here */
-			time.Sleep(20 * time.Millisecond)
+		//	time.Sleep(20 * time.Millisecond)
+			time.Sleep(1 * time.Millisecond)
 			sx.txRaw([]byte{4, 20})
 		}
 
@@ -287,7 +288,8 @@ func (sx *SerialXport) Tx(bytes []byte) error {
 		 * carriage return (and possibly LF 2 bytes), */
 
 		/* all totaled, MTU-4 bytes bytes should work */
-		writeLen := util.Min(sx.cfg.Mtu - 4, totlen-written)
+		writeLen := util.Min(1000, totlen-written)
+	//	writeLen := util.Min(sx.cfg.Mtu - 4, totlen-written)
 
 		writeBytes := base64Data[written : written+writeLen]
 		sx.txRaw(writeBytes)
